index.rst

```diff
@@ -1,42 +1,621 @@
+
+Unfortunately, a general algorithm for computing the decorrelation factorization does not yet exist, making decorrelated coadds a mostly theoretical concept at present.  Some initial steps towards developing such an algorithm will be discussed in later sections.
+
+
+Kaiser Coadds
+-------------
+
+If the input images to a likelihood coadd meet certain restrictive conditions, an algorithm developed by [Kaiser2001]_ (and rediscovered by [Zackay2015]_) can be used to build decorrelated coadd.  These conditions include:
+
+- The noise in the input images must be white and uncorrelated.
+- The PSFs of the input images must (individually) be spatially constant.
+- The input images have no missing pixels, and the coadd area does not include any boundaries where the number of input images changes.
+
+Under these conditions, :math:`\boldsymbol{\Phi}` has no spatial variation, giving it a particularly simple form in Fourier space:
+
+.. math::
+  \tilde{\Phi}({\bf u}, {\bf v})
+    = \delta({\bf u}, {\bf v}) \sum_i \frac{
+        \tilde{\phi}_i({\bf u}) \, \tilde{\phi}_i({\bf v})
+      }{
+        C_i
+      }
+    = \delta({\bf u}, {\bf v}) \sum_i \frac{
+        \left|\tilde{\phi}_i({\bf u})\right|^2
+      }{
+        C_i
+      }
+
+(recall that :math:`C_i` is now just a scalar, as the variance is constant and there is no covariance).  Recognizing that the matrix products in :eq:`eq:decorrelated_factorization` are just convolutions when the products are spatially constant, the Fourier-space equivalent for Kaiser coadds is
+
+.. math::
+  \tilde{\Phi}({\bf u}, {\bf v}) =
+    \tilde{\phi}_\mathrm{ksr}({\bf u})
+    \,
+    \left[ C_\mathrm{ksr}^{-1} \, \delta({\bf u}, {\bf v}) \right]
+    \,
+    \tilde{\phi}_\mathrm{ksr}({\bf v})
+
+The solution is trivial (and unique, assuming a normalized PSF):
+
+.. math::
+  \tilde{\phi}_\mathrm{ksr}({\bf u}) =&
+    \sqrt{
+      \frac{
+        \sum_i \left|\tilde{\phi}_i({\bf u})\right|^2 \, C_i^{-1}
+      }{
+        \sum_i C_i^{-1}
+      }
+    }\\
+  C_\mathrm{ksr} =& \frac{1}{\sum_i C_i^{-1}}
+
+The equivalent for :math:`\boldsymbol{\Psi}` and :eq:`eq:decorrelated_coadd` is
+
+.. math::
+  \tilde{\Psi}({\bf u}) = \sum_i
+    \frac{
+      \tilde{\phi}^*_i({\bf u}) \, \tilde{z}({\bf u})
+    } {
+      C_i
+    }
+  =
+  \tilde{\phi}_\mathrm{ksr}({\bf u}) \,
+    C_\mathrm{ksr}^{-1} \, \tilde{z}_\mathrm{ksr}({\bf u})
+
+with solution
+
+.. math::
+  \tilde{z}_\mathrm{ksr}({\bf u}) =
+    \frac{
+      \sum_i \tilde{\phi}^*_i({\bf u}) \, \tilde{z}_i({\bf u}) \, C_i^{-1}
+    }{
+      \sqrt{
+        \sum_i \left| \tilde{\phi}_i({\bf u}) \right|^2 \, C_i^{-1}
```

Comment:
Good catch again.  I was confused because the formula as it currently stands matches Zackay and Ofek's Eqn. 7, but that's only because they've redefined their coadd flux units so that the coadd has unit variance.